tests/dapr_tests.mk

```diff
@@ -155,6 +155,8 @@ setup-app-configurations:
 # Apply component yaml for state, secrets, pubsub, and bindings
 setup-test-components:
 	$(KUBECTL) apply -f ./tests/config/kubernetes_secret.yaml --namespace $(DAPR_TEST_NAMESPACE)
+	$(KUBECTL) apply -f ./tests/config/kubernetes_secret_2.yaml --namespace $(DAPR_TEST_NAMESPACE)
```

Comment:
Can we merge kuberntes_secret.yaml with kubernetes_secret_2.yaml ? maybe we can use `---` delimiter to combine both into single yaml.